pkg/thumbnails/thumbnails.go

```diff
@@ -69,16 +69,15 @@ func (s SimpleManager) GetStored(ctx Context) []byte {
 }
 
 func (s SimpleManager) generate(ctx Context, img image.Image) image.Image {
-	thumbnail := resize.Thumbnail(uint(ctx.Width), uint(ctx.Height), img, resize.Lanczos2)
+	thumbnail := resize.Thumbnail(uint(ctx.Resolution.Width), uint(ctx.Resolution.Height), img, resize.Lanczos2)
```

Comment:
Is it good practice to pass domain-values inside context?

Reply:
Currently it is a custom context defined in this package.
Probably context is a bad name but I couldn't think of a fitting one.

It's an object that combines different values needed to fullfill a thumbnail request.
Maybe I should rename it to Request? :D

Reply:
Ah :bulb:  If it is a custom struct then it makes sense to rename it to something else to reduce confusion with golang contexts.